.devcontainer/devcontainer.json

```diff
@@ -73,5 +73,9 @@
     }
   },
   // Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-  "remoteUser": "vscode"
+  "remoteUser": "vscode",
+  // This is running the same command as the VSCode Task 'devcontainer: rebuild oh-my-posh'
+  // It Compiles *oh-my-posh* from this repo while **overwriting** your preinstalled stable release.'
+  // Ideal for getting straight into developing & testing whilst using a devcontainer
+  "postCreateCommand": "cd src && go build -v -o /home/vscode/bin/oh-my-posh -ldflags \"-s -w -X 'github.com/jandedobbeleer/oh-my-posh/src/build.Version=development-$(git --no-pager log -1 --pretty=%h-%s)' -extldflags '-static'\""
```

Comment:
doesn't this imply we can remove downloading oh-my-posh on container creation? Currently we do it there.

Reply:
I wanted to keep it simple and do the same command the VSCode task runs, but yes perhaps that part could be removed, but I didn't want to change too much in case it broke things

Reply:
@warrenbuckley you can always try and remove it, rebuild without cache and see if everything starts correctly. If so, delete away!

Reply:
Will give it a try later on

Reply:
@JanDeDobbeleer give it a go and see what you think :)

Reply:
there was one issue